src/comitNode.ts

```diff
@@ -32,6 +32,10 @@ export interface Swap {
   actions: Action[];
 }
 
+export interface ComitMetaData {
```

Comment:
```suggestion
export interface ComitMetadata {
```
metadata is one word :)